src/components/TestStringOnAutomata.ts

```diff
@@ -5,6 +5,15 @@ export function testStringOnAutomata(testString: string): string {
   let myDFA = StateManager.dfa;
   let runner = new DFARunner(myDFA, testString.split(""));
   runner.runUntilConclusion();
+  if (StateManager.checkDebug()) {
+    let runnerdebug = new DFARunner(myDFA, testString.split(""));
+
+    let path = runnerdebug.runUntilConclusionDebug();
```

Comment:
`runUntilConclusionDebug()` doesn't exist on `DFARunner`, so the code won't compile.

Rather than trying to add a new method to `DFARunner`, it may be wiser to just use the `DFARunner.runStep()` method. [`runUntilConclusion()` itself simply calls `runStep()` continuously until the DFA has finished](https://github.com/AutomatonBuilderGUI/automaton-ts/blob/f15ae18026c19368da1f264ee065cefdf7b51989/src/dfa/DFARunner.ts#L164):
```ts
  runUntilConclusion() {
    while (!this.hasConcluded()) {
      this.runStep();
    }
  }
```
So if you want to run a DFA one step at a time, you can call `runStep()` on it yourself, and store the result of each call in a stack to go backwards if need be.